manager/integration/tests/run.sh

```diff
@@ -6,6 +6,10 @@ if [[ $@ =~ "--junitxml=" ]] ; then
   py.test -v . $@  >> /tmp/longhorn-pytest 2>&1
   
   cat ${LONGHORN_JUNIT_REPORT_PATH}
+
+elif [[ $@ =~ "test_stress" ]] ; then
```

Comment:
Is it possible to run the stress test with other tests? Not sure why we need a separate way to run it. It's for a small scale and should pass every night.

Reply:
We need to run stress test only in parallel with `pytest-xdist`, so we have to run the test in separate step.